### PR TITLE
vectorize-pixelart: add page

### PR DESCRIPTION
**Pull request by Yutyo**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/vectorize-pixelart.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# vectorize-pixelart
+
+> Convert raster pixel art graphics to vector.
+> More information: <https://github.com/und3f/vectorize-pixelart>
```
Comment (bl-ue) [inline pages/common/vectorize-pixelart.md:4]:
```suggestion
> More information: <https://github.com/und3f/vectorize-pixelart>.
```

**Comment by bl-ue** [comment]
💡  @Yutyo the PR title format for adding new page is `page-name: add page`

**Comment by Yutyo** [comment]
🦗

File: pages/common/vectorize-pixelart.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# vectorize-pixelart
+
+> Convert raster pixel art graphics to vector.
+> More information: <https://github.com/und3f/vectorize-pixelart>.
+
+- Vectorize a PNG image to an SVG vector:
+
+`vectorize-pixelart {{image.png}} {{output.svg}}`
+
+- Vectorize to an EPS vector:
```
Comment (bl-ue) [inline pages/common/vectorize-pixelart.md:10]:
```suggestion
- Convert a PNG image to an EPS image:
```

File: pages/common/vectorize-pixelart.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# vectorize-pixelart
+
+> Convert raster pixel art graphics to vector.
```
Comment (bl-ue) [inline pages/common/vectorize-pixelart.md:3]:
```suggestion
> Convert PNG pixel art graphics to SVG/EPS vector images.
```

**Review by bl-ue — CHANGES_REQUESTED** [review]

File: pages/common/vectorize-pixelart.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+> More information: <https://github.com/und3f/vectorize-pixelart>.
+
+- Vectorize a PNG image to an SVG vector:
+
+`vectorize-pixelart {{image.png}} {{output.svg}}`
+
+- Vectorize to an EPS vector:
+
+`vectorize-pixelart {{image.png}} {{output.eps}}`
```
Comment (owenvoke) [inline pages/common/vectorize-pixelart.md:12]: To be fair, as both of these do the same thing (just with different output formats) and don't have any flags, we can probably do a single example with something like the following:
```suggestion
- Convert a PNG image to a vector image format:

`vectorize-pixelart {{path/to/input.png}} {{path/to/output.svg|eps}}`
```

Comment (bl-ue) [reply pages/common/vectorize-pixelart.md:12]: What about `{{path/to/output.svg|.eps}}` (with `|.eps`)?

**Review by owenvoke — APPROVED** [review]

File: pages/common/vectorize-pixelart.md (hunk 4 above)
Comment (owenvoke) [reply pages/common/vectorize-pixelart.md]: I'm not entirely sure what our standard is for multiple extensions.

**Review by einverne — APPROVED** [review]

File: pages/common/vectorize-pixelart.md [hunk 5]
```diff
@@ -0,0 +1,8 @@
+# vectorize-pixelart
+
+> Convert PNG pixel art graphics to SVG/EPS vector images.
+> More information: <https://github.com/und3f/vectorize-pixelart>.
+
+- Convert a PNG image to a vector image format:
+
+`vectorize-pixelart {{path/to/input.png}} {{path/to/output.svg|eps}}`
```
Comment (bl-ue) [inline pages/common/vectorize-pixelart.md:8]: Let's just do this for now.  Someone can come a long and discuss this for the long run later.
```suggestion
`vectorize-pixelart {{path/to/input.png}} {{path/to/output.svg|.eps}}`
```

**Review by bl-ue — APPROVED** [review]
Thank you @Yutyo 🙂

**Pull request merged**